modules/parse_dmc_classification.nf

```diff
@@ -1,25 +1,15 @@
 process PARSE_DMC_CLASSIFICATION {
   label 'light'
-  publishDir "${outdir}", mode: 'copy'
 
```

Comment:
Have been given a suggestion to explicitly specify python containers here... 
    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
        'https://depot.galaxyproject.org/singularity/biopython:1.75':
        'quay.io/biocontainers/biopython:1.75' }"

Reply:
That's a nice suggestion thanks, I'll add it !